enterprise/server/remote_execution/blockio/blockio.go

```diff
@@ -458,6 +549,31 @@ func IsEmptyOrAllZero(data []byte) bool {
 	return true
 }
 
+// expandHoles inserts explicit holes for any chunk start offsets not explicitly
+// represented in the given list. The input chunks are expected to be sorted in
+// increasing order of offset.
+func expandHoles(chunks []*Chunk, chunkSize, totalSize int64) []Store {
```

Comment:
the holes idea feels a little premature and is adding a decent amount of complexity -- could we remove holes for now / treat all chunks the same and add them back later if necessary?